R/mod_metricBox.R

```diff
@@ -122,11 +132,18 @@ metricBoxServer <- function(id, title, desc, value, score = "NULL",
             )
           ),
           div(class = "card-footer bg-transparent", desc)
-        )
-      )
-      
-      
+        ))
+     
       
+      if (title %in% c("Dependencies","Reverse Dependencies")){ 
+         html_component <- shiny::tagAppendAttributes(html_component, 
+                                                      onMouseOver="this.style['box-shadow'] = '2px 2px 2px black';
+                                                      this.style['cursor'] = 'pointer'",
+                                    onMouseOut="this.style['box-shadow'] = '2px 2px 2px white'")
```

Comment:
```suggestion
                                                      onMouseOver="this.style['box-shadow'] = '2px 2px 2px var(--bs-info)';
                                                      this.style['cursor'] = 'pointer'",
                                    onMouseOut="this.style['box-shadow'] = 'none'")
```

Reply:
Mostly a fan of this. I don't exactly like the black shadow since the border is a different color but my implementation above doesn't really look great either because it needs to be darker.

I would suggest removing the border on the mouse out event though.